docs/permission/getting-started.md

```diff
@@ -34,6 +34,10 @@ Like many other parts of Backstage, the permissions framework relies on informat
 
 [The IdentityResolver docs](../auth/identity-resolver.md) describe the process for resolving group membership on sign in.
 
+## Optionally add cookie-based authentication
```

Comment:
Do you think it's worth gathering these types of sections and starting a new "Advanced" page? I am cognizant of keeping these pages as focused on basic tutorial as possible.

Reply:
I'm not sure if we should consider this one advanced, since most users will have techdocs installed and will need to implement this as soon as they enable catalog permissions. So I'd consider it more of a prerequisite than part of the actual tutorial

Reply:
👍 for the techdocs case